library/core/class.dispatcher.php

```diff
@@ -796,6 +797,13 @@ private function _mapParts($Parts, $ControllerKey) {
             }
         }
     }
+
+    /**
+     * Trigger an event allowing addons to modify response headers.
+     */
+    public function sendHeaders() {
```

Comment:
Registering as a callback with `header_register_callback` appears to call the function externally.  Switching up its visibility from public causes errors.